### ISSUE TITLE
"Invalid input packet" with new 5.6.6  on raspberry pi

### ISSUE THREAD

**Issue by Nicolas-N** [dashingsoft/pyarmor#129]
Hello,

I use successfully pyarmor 5.5.4 to obfuscate and run an very simple script on raspberri pi 3 ( raspbian stretch/ python 3.5).

When I try with pyarmor 5.6.6, I get :
```
Invalid input packet.
Check license failed.
```

I've checked license.lic, pytransform.key and _pytransform.so in dist directory: they are created by the pyarmor obfuscate phase.

From fresh install, with 5.6.6 trial version,  "pyarmor register" show:
```
INFO     Getting remote file: https://github.com/dashingsoft/pyarmor-core/raw/v5.5.2/platforms/index.json
INFO     Search library for linux32 and arch armv7l
INFO     Loading platforms information
INFO     Found available library armv7/_pytransform.so
INFO     Downloading library file ...
INFO     Getting remote file: https://github.com/dashingsoft/pyarmor-core/raw/v5.5.2/platforms/armv7/_pytransform.so
INFO     Create target path: /home/pi/env/lib/python3.5/site-packages/pyarmor/platforms/linux32/armv7l
INFO     Writing target file: /home/pi/env/lib/python3.5/site-packages/pyarmor/platforms/linux32/armv7l/_pytransform.so
INFO     Download pytransform library file successfully.
INFO     Bootstrap OK.
INFO     PyArmor Trial Version 5.6.6
PyArmor Trial Version 5.6.6
```
So I suppose there's no change in _pytransform.so since 5.5.2. 
On this install: In the dist directory the license.lic file is the default one and belong to the one in public_capsule.zip (which seems ok) and the .pyarmor_capsule.zip in my home dir also belong to the one in public_capsule.zip.
I don't understand what I do wrong with the 5.6.6 version, maybe it's a bug on the new pyarmor raspberry pi version.

Moreover, I've recently purchase a registration keyfile since I fall back to the 5.5.4 version and register the capsule and the license.lic manually. 
I've get the obfuscate script working but the license.lic file in the dist directory is the default one (starting with HERhc2hpbmdz ...). I wonder if it's not suppose to be the one I get in the pyarmor-regfile-1.zip.

Any help will be appreciated

**Comment by jondy** [comment]
There are some changes since _pytransform.so 5.5.1 for armv7. So how about using `_pytransform.so` 5.5.0 like this:
```
wget https://github.com/dashingsoft/pyarmor-core/raw/v5.5.0/platforms/armv7/_pytransform.so
mv _pytransform.so /home/pi/env/lib/python3.5/site-packages/pyarmor/platforms/linux32/armv7l

pyarmor register pyarmor-regfile-1.zip
pyarmor obfuscate foo.py
cd dist
python foo.py

```

In version 5.5.4, after register the capsule and license.lic manually, it's better to run command `touch ~/.pyamor_capsule.zip` to be sure it's newer than the `license.lic`.  Otherwise pyarmor (version <5.6.0) will generate a new capsule in bootstrap

**Comment by jondy** [comment]
There is a bug with v5.6.6 on raspberry pi (armv7), it has been fixed in v5.6.7. Please upgrade to the latest version to fix this issue.

**Comment by Nicolas-N** [comment]
Upgrade to 5.6.7 correct the problem.
Everything work fine now.
Thanks.
